bin/fix_my_workspace

```diff
@@ -4,8 +4,17 @@ $VERBOSE = nil  # Don't care about world writable dir warnings and the like
 if $_fix_my_workspace_version_check
   $_fix_my_workspace_version = '0.5.0'
 else
-  require 'origen'
-
+  # if we're running this from a development environment (i.e., bin/ inside of origen repo) then
```

Comment:
I updated my Origen to use the latest updater but it still didn't work. The error I'm getting is:

<code>
/pkg/fs-ruby-/2.3.0/x86_64-linux/lib/ruby/gems/2.3.0/gems/origen-0.11.0/lib/origen.rb:280:in `root': Something went wrong resolving the application root! (RuntimeError)
</code>

Does this need to be fixed back at that install of Origen? That's not the version I'm sourcing either. But the Origen.root issue is something that I've corrected in this PR. Doesn't seem to be being used by the fix_my_workspace unless I explicitly load it, as in the changes above.

It seems like, since that's a standalone script, its using the Origen version installed in the system gems, not the version I'm pointing it to.